src/vsr/grid_scrubber.zig

```diff
@@ -79,7 +79,7 @@ pub fn GridScrubberType(comptime Forest: type) type {
             done: bool,
 
             /// "next" belongs to the FIFOs.
```

Comment:
Should we update the "next" in this comment?

Reply:
Definitely. We actually might not need these comments anymore, as now we spell the type of the thing the link belongs to, but I think extra clarity won't hurt!